backend/src/ee/services/secret-rotation/secret-rotation-service.ts

```diff
@@ -97,15 +97,16 @@ export const secretRotationServiceFactory = ({
     if (!folder) throw new NotFoundError({ message: "Secret path not found" });
     ForbiddenError.from(permission).throwUnlessCan(
       ProjectPermissionActions.Edit,
-      subject(ProjectPermissionSub.Secrets, { environment, secretPath })
+      subject(ProjectPermissionSub.Secrets, { environment, secretPath, secretName: "", secretTags: [] })
```

Comment:
Hmm something is telling me secret rotations should also have their own subjects because `secretName: "", secretTags: []` also doesn't make sense to compare in this context

Reply:
I guess on revamping secret rotation let's address this

Reply:
Todo: Make `secretName: "", secretTags: []` optional so we don't have to set empty values